GDD.md

```diff
@@ -11,25 +15,50 @@ This runs on Windows, Linux, and in Web.
 
 ## Development Software
 Programming: Godot Engine
+
 Art: 
+
 Music: 
```

Comment:
ill get back to you on this one. I've got a producer friend he said he might help me direct the music

Reply:
maybe the music is moody when everything is grayscaled
and gets happier once the player gets the flowers